test/truefi2/TrueLender2.test.ts

```diff
@@ -133,13 +130,12 @@ describe('TrueLender2', () => {
     await tru.approve(stkTru.address, parseTRU(15e6))
     await stkTru.stake(parseTRU(15e6))
     await timeTravel(1)
-
     loan1 = await createLoan(loanFactory, borrower, pool1, 100000, YEAR, 100)
 
     loan2 = await createLoan(loanFactory, borrower, pool2, 500000, YEAR, 1000)
 
-    await rateAdjuster.addPoolToTVL(pool1.address)
-    await rateAdjuster.addPoolToTVL(pool2.address)
+    await poolFactory.supportPool(pool1.address)
+    await poolFactory.supportPool(pool2.address)
```

Comment:
We already call `poolFactory.supportPool()` in lines 109-110 above. I think this might have come from a git merge.